test/integration/targets/subversion/roles/subversion/tasks/setup.yml

```diff
@@ -11,6 +11,11 @@
   package:
     name: '{{ subversion_packages }}'
     state: present
+  when: ansible_distribution != 'Alpine'
+
+- name: install SVN pre-reqs - Alpine
+  command: 'apk add -U {{ subversion_packages|join(" ") }}'
```

Comment:
We don't have the `apk` module here, and as far as tests are concerned, I think this is sufficient.